doc/index.rst

```diff
@@ -40,7 +40,11 @@ Each mapping rule can have one of three types:
 
 3. A field mapping rule is defined by the attributes of a message mapping rule and:
 
-   - a dictionary ``fields_1_to_2`` mapping ROS 1 field names to ROS 2 field names.
+   - a dictionary ``fields_1_to_2`` mapping ROS 1 field selections to ROS 2 field selections. A field selection
```

Comment:
Each Sentence should start on a new line (see https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#markdown-restructured-text-docblocks).